src/endpoints/v1.es6.js

```diff
@@ -829,13 +829,13 @@ class APIv1Endpoint {
 
       formatBody: function(options) {
         return function(body) {
-          if (body.data && body.data.children && body.data.children[0]) {
+          if (has(body, 'data.children.0')) {
             if (options.id) {
               return new Link(body.data.children[0].data).toJSON();
             } else {
               return body.data.children.map(c => new Link(c.data).toJSON());
             }
-          } else if (body.data.children) {
+          } else if (has(body, 'data.children')) {
```

Comment:
looking at this again I should probably get rid of the else and just check for data.children above.